routes/handlers/entryPoint.js

```diff
@@ -0,0 +1,17 @@
+const osdiLinks = require('../../lib/links.js');
+
+exports.get = {
+
+  handler: function (req, reply) {
+    reply({
+      motd: 'Welcome to the Resistance Calendar API Entry Point',
+      max_pagesize: 25,
```

Comment:
Can we add this number to the config and use also in the [events.js#7](https://github.com/ResistanceCalendar/resistance-calendar-api/blob/master/routes/handlers/event.js#L7)